README.md

```diff
@@ -1,399 +1,394 @@
+* Version should be pinned in the root module where the modules are called from using the `ref` argument:
 
       module "bootstrap_bucket" {
         source          = "spring.paloaltonetworks.com/mekanayake/terraform-aws-panfw-bootstrap?ref=0.1.0"
         bucket_prefix   = "GlobalProtect-bootstrap-"
         local_directory = "vmseries-bootstrap-package"
       }
 
-## `2. Coding Practices`
+## 2. Coding Practices
 
-### `2.1 Variables`
+### 2.1 Variables
 
 * Static values shouldn't be hardcoded inside the Terraform configuration, static values should be defined as Terraform variables, this extends the flexibility of the Terraform nodules for future use.
 
-  > BAD Example - All the arguments provided to the `contains` function are hardcoded
+  * BAD example, all the arguments provided to the `contains()` function are hardcoded
 
-      contains(["169.254.0.0/28", "169.254.1.0/28", "169.254.2.0/28", "169.254.3.0/28", "169.254.4.0/28", "169.254.5.0/28", "169.254.169.240/28"], "169.254.1.0/28")
+        contains(["169.254.0.0/28", "169.254.1.0/28", "169.254.2.0/28", "169.254.3.0/28", "169.254.4.0/28", "169.254.5.0/28", "169.254.169.240/28"], "169.254.1.0/28")
 
-  > GOOD Example - All the arguments provided to the `contains` function are extracted and defined as variables
+  * GOOD example, all the arguments provided to the `contains()` function are extracted and defined as variables
 
-      variable "exceptionList" {
-        default = ["169.254.0.0/28", "169.254.1.0/28", "169.254.2.0/28", "169.254.3.0/28", "169.254.4.0/28", "169.254.5.0/28", "169.254.169.240/28"]
-      }
+        variable "exceptionList" {
+          default = ["169.254.0.0/28", "169.254.1.0/28", "169.254.2.0/28", "169.254.3.0/28", "169.254.4.0/28", "169.254.5.0/28", "169.254.169.240/28"]
+        }
 
-      variable "tunnelCidrRange" {
-        default = "169.254.1.0/28"
-      }
+        variable "tunnelCidrRange" {
+          default = "169.254.1.0/28"
+        }
 
-      contains(var.exceptionList, var.tunnelCidrRange)
+        contains(var.exceptionList, var.tunnelCidrRange)
 
 * Optional variables for resources should still be included (where sensible) to ensure future extendability of the module
 
-  > Example - `ipv6_cidr_block` is optional, however the optional argument is included and the value is provided with the help of `lookup` function to ensure future extensibility
+  * GOOD example, `ipv6_cidr_block` is optional, however the optional argument is included and the value is provided with the help of `lookup` function to ensure future extensibility
 
-      resource "aws_subnet" "in_secondary_cidr" {
-        vpc_id     = vpc_id
-        cidr_block = each.value.cidr_block
-        ipv6_cidr_block = lookup(each.value, "ipv6_cidr_block", null)
-      }
+        resource "aws_subnet" "in_secondary_cidr" {
+          vpc_id     = vpc_id
+          cidr_block = each.value.cidr_block
+          ipv6_cidr_block = lookup(each.value, "ipv6_cidr_block", null)
```

Comment:
Nit: fix formatting so the equal signs in the key-value pair are one under another?

Reply:
duh, thanks for catching this 👍